### PR TITLE
Added Graph Widget/Exercise Widget

### PR DESCRIPTION
**Pull request by Dez1999**
Holistic Health Graph Widget

### CODE REVIEW

File: packages/graph/stories/Test.stories.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
```
Comment (hamzahalilovic) [inline packages/graph/stories/Test.stories.js:1]: You can delete this file

File: packages/graph/stories/App.stories.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, { useState, useEffect } from "react";
+import App from "../src/graph";
+
+export default { title: "Test" };
```
Comment (hamzahalilovic) [inline packages/graph/stories/App.stories.js:4]: you can change the title to "Graph"

**Review by Dez1999 — COMMENTED** [review]
Deleted test.stories.js file and changed title to Graph

**Comment by Dez1999** [comment]
Resolved conflicts in package.json and yarn.lock

**Pull request closed without merging**